README.md

```diff
@@ -287,6 +287,100 @@ However, this will rebuild entire VM, which means that besides your own scripts,
 $ capstan package compose --update node-micro/tutorial
 ```
 
+## Deploying Unikernels on Kubernetes
+You can deploy `osv-microservice-demo` application on Kubernetes at ease. If you don't have a working Kubernetes cluster up-and-running with [Virtlet](https://github.com/Mirantis/virtlet) plugin installed,
```

Comment:
I noticed that most of the other text is properly wrapped - you've probably forgotten to format this line?